### DISCUSSION TITLE
萌新提问怎么升级呀

### DISCUSSION THREAD

**Discussion by Elysia888** [share121/inter-knot#2885] [其他]
![1722779975355](https://github.com/user-attachments/assets/aa3ff42b-ca47-4370-a17d-2449431e1966)
求知

**Comment by NiuBoss123** [comment]
那个等级和你账号下有多少个仓库有关（目前来看是这样，因为我账号下好像正好是11个仓库）

**Comment by FRIKY-c** [comment]
等级应该是看你github账号的吧
